content/docs/learn/typed-errors/index.md

```diff
@@ -7,6 +7,29 @@ import DocCardList from '@theme/DocCardList';
 
 # <decorated-text icon={useCurrentSidebarCategory().customProps.icon} title={frontMatter.title} />
 
-<p>{useCurrentSidebarCategory().customProps.description}</p>
+_Typed errors_ refer to a technique from functional programming in which we
+make _explicit_ in the signature (or _type_) the potential errors that may
+arise during the execution of a piece of code. Arrow provides two different
+**approaches** to typed errors:
+
+1. The `Raise` DSL uses an extension receiver which represents a _context_
```

Comment:
So... is there a `Raise` class?  I keep looking for one, since it's capitalized, but I don't see it.  I see `raise`, the function/keyword.  Perhaps it's just my newness to Kotlin, but referring to the DSL as though it were a class name when it's named for a function name used in only one part of the API is... confusing.

Reply:
My 2c... 
There is a `Raise` interface which provides the Raise DSL to all code blocks executing in the Raise context. `either { }` is a function of type `fun <E> either(block: Raise<E>.() -> Unit)`, i.e. the code in your either has access to this DSL. `raise` is a function provided in this DSL. Does that make sense? :)

Reply:
Added a new block discussing this matter, and where the name originates from